src/auth/login/login.js

```diff
@@ -55,40 +57,24 @@ class Login extends Component {
                 ? "Or Sign In with"
                 : "Or SignUp with"}
             </p>
-            <Row>
-              <Col className="button-column">
+            <div className="login__options">
+              <div className="">
                 <a
-                  href="http://localhost:4000/auth/google"
+                  href="http://localhost:5000/auth/google"
                   style={{ padding: "1vh" }}
                 >
-                  <Button
-                    className="selectorbtn"
-                    type="submit"
-                    variant="primary"
-                    color="primary"
-                    size="sm"
-                  >
-                    <span className="selectorbtn-content">Google</span>
-                  </Button>
+                  <img src={GoogleLogin} alt="Google" className="google__login"/>
                 </a>
-              </Col>
-              <Col className="button-column">
+              </div>
+              <div className="" >
                 <a
-                  href="http://localhost:4000/auth/github"
+                  href="http://localhost:5000/auth/github"
```

Comment:
.

Reply:
@vaibhavdaren As the GitHub and google login is not implemented yet, so it's there from the initial phase.
When I will implement the google and GitHub login then will integrate that.